src/mpi/mpi_caf.c

```diff
@@ -429,6 +429,35 @@ caf_runtime_error (const char *message, ...)
   exit(EXIT_FAILURE);
 }
 
+/* Error handling is similar everytime. Keep in sync with single.c, too. */
+static void
+caf_internal_error (const char *msg, int *stat, char *errmsg,
+		    size_t errmsg_len, ...)
+{
+  va_list args;
+  va_start (args, errmsg_len);
+  if (stat)
+    {
+      *stat = 1;
+      if (errmsg_len > 0)
+	{
+	  int len = snprintf (errmsg, errmsg_len, msg, args);
+	  if (len >= 0 && errmsg_len > (size_t) len)
+	    memset (&errmsg[len], ' ', errmsg_len - len);
+	}
+      va_end (args);
+      return;
+    }
+  else
+  {
+    fprintf(stderr, "Fortran runtime error on image %d: ", caf_this_image);
```

Comment:
@vehre how about changing "Fortran runtime error" to "OpenCoarrays internal error" to be consistent with the name `caf_internal_error` and to be more specific about the location of the error as being in OpenCoarrays rather than in the `libgfortran` runtime library.

Reply:
I just pushed this change.

Reply:
That's fine. I just copied the code from gfortran not reflecting too much that we are in OpenCoarrays.